app/utils/sentry.js

```diff
@@ -0,0 +1,32 @@
+const Sentry = require('@sentry/node')
+
+function initialiseSentry () {
+  Sentry.init({
+    dsn: process.env.SENTRY_DSN,
+    environment: process.env.ENVIRONMENT,
+    beforeSend (event) {
+      if (event.request) {
+        delete event.request // This can include sensitive data such as card numbers
+      }
+      return event
+    }
+  })
+  return Sentry
+}
+
+const addSentryToErrorLevel = (originalLogger) => {
+  const sentryLogger = Object.create(originalLogger)
+  sentryLogger.error = function (msg) {
```

Comment:
I think this should either be an arrow function or a proper named function. As you have it, it is an old skool anonymous function. I think an arrow function is fine here

Reply:
ah good spot 👍 I've changed to an arrow function...